ManatiCR/manatime
# 4: Deactivate clients.

**Steps to test**
- Create a new client node.
- Check if the new VBO action is executed successfully.

**DoD**
- In clients view (created in a previous ticket) ensure we have a VBO action to enable/disable clients.

[issue](https://github.com/ManatiCR/manatime/issues/4)
